kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala

```diff
@@ -34,12 +34,14 @@ import org.apache.kyuubi.metrics.{MetricsConf, MetricsSystem}
 import org.apache.kyuubi.server.metadata.jdbc.JDBCMetadataStoreConf
 import org.apache.kyuubi.service.{AbstractBackendService, AbstractFrontendService, Serverable, ServiceState}
 import org.apache.kyuubi.session.KyuubiSessionManager
-import org.apache.kyuubi.util.{KyuubiHadoopUtils, SignalRegister}
+import org.apache.kyuubi.util.{KyuubiHadoopUtils, SemanticVersion, SignalRegister}
+import org.apache.kyuubi.util.ScalaVersionUtils._
 import org.apache.kyuubi.zookeeper.EmbeddedZookeeper
 
 object KyuubiServer extends Logging {
   private[kyuubi] var kyuubiServer: KyuubiServer = _
   @volatile private[kyuubi] var hadoopConf: Configuration = _
+  lazy val serverRuntimeScalaVersion: SemanticVersion = scalaRuntimeSemanticVersion
```

Comment:
seems we don't need to introduce this variable, simply call `SCALA_COMPILE_VERSION` is sufficient?